src/framework/global/thirdparty/deto_async/async/internal/abstractinvoker.cpp

```diff
@@ -192,3 +198,16 @@ void AbstractInvoker::removeQInvoker(QInvoker* qi)
     std::lock_guard<std::mutex> lock(m_qInvokersMutex);
     m_qInvokers.remove(qi);
 }
+
+bool AbstractInvoker::containsReceiver(Asyncable* receiver) const
```

Comment:
Feels like this can hit the performance?

One other thing, I thought we're removing the necessary callbacks at destruction point of a receiver? @igorkorsukov FYI

Reply:
I implemented this solution after discussing with Igor, we came to the conclusion that this is the best solution at the moment. Usually, an Invoker doesn't have many receivers.

> One other thing, I thought we're removing the necessary callbacks at destruction point of a receiver?

Yes, we remove all callbacks in the destructor of the receiver. The problem is that we delete the receiver after passing this receiver to the Invoker. At the time of the creation of the Invoker, the receiver was valid; when the callback was called, the receiver is destroyed.